### PR TITLE
NO-ISSUE: added deploy-rabbitmq

### PR DESCRIPTION
**Pull request by amir-yogev-gh**
Added the option to deploy local RabbitMQ.
Usage:
make deploy-rabbitmq

### CODE REVIEW

File: deploy/deploy.mk [hunk 1]
```diff
@@ -28,6 +28,10 @@ deploy-db:
 	podman exec -it flightctl-db psql -c 'ALTER ROLE admin WITH SUPERUSER'
 	podman exec -it flightctl-db createdb admin || true
 
+deploy-rabbitmq:
+	podman rm -f flightctl-rabbitmq || true
+	cd deploy/podman && podman-compose up -d flightctl-rabbitmq
+
 kill-db:
```
Comment (hexfusion) [inline deploy/deploy.mk:35]: We also want a way to tear down the message queue. `kill-rabbitmq` WDYT?

**Review by hexfusion — COMMENTED** [review]
Looks reasonable just one nit, thanks for the PR!

File: deploy/deploy.mk [hunk 2]
```diff
@@ -28,6 +28,10 @@ deploy-db:
 	podman exec -it flightctl-db psql -c 'ALTER ROLE admin WITH SUPERUSER'
 	podman exec -it flightctl-db createdb admin || true
 
+deploy-rabbitmq:
```
Comment (avishayt) [inline deploy/deploy.mk:31]: Let's call it "deploy-mq".  Rabbitmq is an implementation detail.  Similar to "deploy-db" instead of "deploy-postgres".

**Review by avishayt — APPROVED** [review]

File: deploy/deploy.mk [hunk 3]
```diff
@@ -28,6 +28,10 @@ deploy-db:
 	podman exec -it flightctl-db psql -c 'ALTER ROLE admin WITH SUPERUSER'
 	podman exec -it flightctl-db createdb admin || true
 
+deploy-mq:
+	podman rm -f flightctl-mq || true
+	cd deploy/podman && podman-compose up -d flightctl-mq
+
```
Comment (avishayt) [inline deploy/deploy.mk:34]: Please add kill-mq

**Pull request closed without merging**